!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  enable: true
  equal: true
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        dictitems:
          Cutting: !!python/object/new:dynamic_reconfigure.encoding.Config
            dictitems:
              groups: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              id: 2
              name: Cutting
              parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              parent: 0
              state: true
              type: ''
              x_enable: false
              x_max: 1100.0
              x_min: 450.0
              y_enable: true
              y_max: 2800.0
              y_min: -100.0
              z_enable: true
              z_max: 2000.0
              z_min: 720.0
            state: []
          Plane_Detection: !!python/object/new:dynamic_reconfigure.encoding.Config
            dictitems:
              groups: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              id: 4
              max_it: 1000
              method: 0
              n_planes: 4
              name: Plane_Detection
              parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              parent: 0
              state: true
              th_dist: 20.0
              type: ''
            state: []
          Subsampling: !!python/object/new:dynamic_reconfigure.encoding.Config
            dictitems:
              enable: true
              equal: true
              groups: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              id: 1
              name: Subsampling
              parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              parent: 0
              state: true
              type: ''
              x: 20.0
              y: 10.0
              z: 10.0
            state: []
          Transformation: !!python/object/new:dynamic_reconfigure.encoding.Config
            dictitems:
              groups: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              id: 3
              name: Transformation
              parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              parent: 0
              rx: 0.0
              ry: 0.0
              rz: 0.0
              state: true
              tx: 0.0
              ty: 0.0
              type: ''
              tz: 0.0
            state: []
        state: []
      id: 0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      type: ''
    state: []
  max_it: 1000
  method: 0
  n_planes: 4
  rx: 0.0
  ry: 0.0
  rz: 0.0
  th_dist: 20.0
  tx: 0.0
  ty: 0.0
  tz: 0.0
  x: 20.0
  x_enable: false
  x_max: 1100.0
  x_min: 450.0
  y: 10.0
  y_enable: true
  y_max: 2800.0
  y_min: -100.0
  z: 10.0
  z_enable: true
  z_max: 2000.0
  z_min: 720.0
state: []
